store/storetest/post_store.go

```diff
@@ -2030,7 +2030,9 @@ func testPostStorePermanentDeleteBatch(t *testing.T, ss store.Store) {
 	o3.CreateAt = 100000
 	o3 = (<-ss.Post().Save(o3)).Data.(*model.Post)
 
-	store.Must(ss.Post().PermanentDeleteBatch(2000, 1000))
+	if _, err := ss.Post().PermanentDeleteBatch(2000, 1000); err != nil {
```

Comment:
Nit: this is totally fine and consistent with the surrounding code but we are trying to use `require.Nil(t,err)` instead.

Reply:
Thank you for this @jespino, I've updated my PR to use the new style